frontend/src/metabase/admin/settings/containers/SettingsEditorApp.jsx

```diff
@@ -193,8 +193,9 @@ class SettingsEditorApp extends Component {
         // HACK - This is used to hide specific items in the sidebar and is currently
         // only used as a way to fake the multi page auth settings pages without
         // requiring a larger refactor.
-        if (section.sidebar === false) {
-          return false;
+        const isNestedSettingPage = Boolean(slug.split("/")[1]);
+        if (isNestedSettingPage) {
+          return null;
```

Comment:
As we don't want to display sub/nesting settings in the sidebar, we can rely on the slug which will always include `/`, and don't have to rely on users passing a special `sidebar` key.